scss/components/_button.scss

```diff
@@ -80,6 +80,10 @@ $button-hollow-hover-lightness: -50% !default;
 /// @type List
 $button-transition: background-color 0.25s ease-out, color 0.25s ease-out !default;
 
+/// Additional responsive classes for .expanded
+/// @type Boolean
+$responsive-expanded-button: true !default;
```

Comment:
@kball true or false

Reply:
I say false

Also - let's rename the variable so it matches the others
$button-responsive-expanded